src/test/java/systemtests/FindCommandSystemTest.java

```diff
@@ -41,6 +40,11 @@ public void find() {
         assertCommandSuccess(command, expectedModel);
         assertSelectedCardUnchanged();
 
+        /* Case: mixed case command word -> rejected */
+        command = "FiNd Meier";
+        //assertCommandFailure(command, MESSAGE_UNKNOWN_COMMAND);
```

Comment:
`/* Case: mixed case command word -> rejected */ ` here should be success? also remember to remove the commented codes